Vienna/Resources/Base.lproj/InfoPlist.strings

```diff
@@ -1 +1,9 @@
-/* Do not remove */
+/* A message that tells the user why Vienna is requesting the ability to send Apple events. */
+"NSAppleEventsUsageDescription" = "";
+
+/* A label for plug-in bundles. This is shown in Finder's Get Info window, for example. */
+"Vienna plug-in" = "Vienna plug-in";
```

Comment:
Shouldn't those lines have the same keys as the info.plist?

Reply:
Nope. The "Plug-in" and "Style" values are non-localisable and only appear in the auto-generated document icons. These keys here are the values for the document description that appear in Finder.